teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization:, name: 'Unknown', age: 0)
+    super(name: name, age: age)
+    @specialization = specialization
```

Comment:
- [ ] I will encourage you to add the parent_permission to the super too since the person class requires it.